src/frontend/src/optimizer/mod.rs

```diff
@@ -1027,6 +1041,12 @@ impl PlanRoot {
             .streaming_use_snapshot_backfill()
     }
 
+    pub fn should_use_cross_db_snapshot_backfill(&self) -> bool {
+        // todo:
+        // self.plan.append_only()
+        self.plan.emit_on_window_close()
```

Comment:
Any reason for this?

Reply:
I was going to add a helper function, but didn't implement or use it later. I will delete it.